### ISSUE TITLE
Tagging

### ISSUE THREAD

**Issue by Cryptoloops** [shadowmoose/RedditDownloader#69]
It would be nice to be able to add your own tags in case nothing in the post is helpful for searching. And perhaps a list of all user-made tags in a list. 

So far i am very impressed with what you have put together. Great work.

**Comment by shadowmoose** [comment]
Interesting. What would you be tagging, specifically? Do you mean tags for individual posts, once they've been downloaded?

**Comment by Cryptoloops** [comment]
Yes, i do mean tagging downloaded posts. For instance, if you sub to a few recipe subreddits and save a bunch of posts, you can add tags to each one individually  (dessert, breakfast, baking, grilling, frying, vegetarian, gluten free, low carb, chicken, pasta, soup, cake, cookies, short cook time...,etc). Then say you want to bake a dessert that uses almond flour and involves chocolate; you can filter all the posts by dessert, then filter those results to show only almond flour, and then folter what’s remaining with chocolate. 

The user would be able to add their own tags to each downloaded post so it would be easily searchable with categories that they decide on.
